packages/terra-clinical-header/src/Header.scss

```
@@ -48,13 +48,14 @@
   }
 
   .title {
-    font-size: 1.285rem;
+    font-size: var(--terra-clinical-header-title-font-size, 1.285rem);
     font-weight: normal;
     height: 100%;
-    line-height: 1.75;
+    line-height: var(--terra-clinical-header-title-line-height, 1.75);
     margin-bottom: 0;
     margin-top: 0;
     overflow: hidden;
+    padding: var(--terra-clinical-header-title-padding, none);
```

`none` is an invalid value for padding - I assume we want 0?
```suggestion
    padding: var(--terra-clinical-header-title-padding, 0);
```